portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -23,42 +23,78 @@
 import javax.servlet.http.HttpServletResponse;
 import java.util.List; 
 import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+ import com.google.appengine.api.datastore.FetchOptions;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** This servlet handles comment's data: 
+ *- doPost() stores get's the recently submitted comment's data from the request 
+ *          and stores in a database using Datastore API
+ *- doGet() returns the comments from the database, aftern converting them to JSON*/
 @WebServlet("/comments-data")
 public class CommentsServlet extends HttpServlet {
-    private List<Comment> comments;
+    private final String commentEntityName = "Comment";
+    private final String firstNameProperty = "firstName";
+    private final String lastNameProperty = "lastName";
+    private final String emailProperty = "email";
+    private final String messageProperty = "message";
+    private final String phoneProperty = "phone";
+    private final String jobTitleProperty = "jobTitle";
```

Comment:
The convention for constants is to also add `static` and upper case separated with underscores. So it will be `private static final String COMMENT_ENTITY_NAME = "Comment"`

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
done